### PR TITLE
Address 0 check

### PR DESCRIPTION
**Pull request by CalabashSquash**
### Motivation and Changes
address(0) checks

### CODE REVIEW

File: contracts/TracerPerpetualSwaps.sol [hunk 1]
```diff
@@ -569,11 +572,25 @@ contract TracerPerpetualSwaps is ITracerPerpetualSwaps, Ownable, SafetyWithdraw
         insurancePoolSwitchStage = _insurancePoolSwitchStage;
     }
 
+<<<<<<< HEAD
```
Comment (kumar-ish) [inline contracts/TracerPerpetualSwaps.sol:575]: Merge conflict here

File: contracts/deployers/PerpsDeployerV1.sol [hunk 2]
```diff
@@ -25,6 +25,9 @@ contract PerpsDeployerV1 is IPerpsDeployer {
             _data,
             (bytes32, address, uint256, address, uint256, uint256, uint256, address, uint256, uint256, uint256)
         );
+        require(_tracerQuoteToken != address(0), "TCRDeploy: _tracerQuoteToken = address(0)");
```
Comment (kumar-ish) [inline contracts/deployers/PerpsDeployerV1.sol]: Over 32 bytes. If you remove the `address` it should be fine?

File: contracts/TracerPerpetualSwaps.sol (hunk 1 above)
Comment (kumar-ish) [inline contracts/TracerPerpetualSwaps.sol]: Merge conflict here

**Review by kumar-ish — CHANGES_REQUESTED** [review]

File: contracts/deployers/PerpsDeployerV1.sol (hunk 2 above)
Comment (kumar-ish) [reply contracts/deployers/PerpsDeployerV1.sol]: Likewise with some of the ones below I think. Not a big deal but probably good practice to truncate them anyways. As an aside, we should talk to Dos about having codes/mappings for different reason messages; e.g. `TCRDeploy: ERR1` corresponds to `TCRDeploy: _tracerQuoteToken can't be the zero address`

**Review by raymogg — APPROVED** [review]

**Review by kumar-ish — APPROVED** [review]

**Pull request merged**